docs/source/create_channel/channel_policies.md

```diff
@@ -0,0 +1,121 @@
+# Channel policies
+
+Channels are a private method of communication between organizations. As a result, most changes to the channel configuration need to be agreed to by other members of the channel. A channel would not be useful if an organization could join the channel and read the data on the ledger without getting the approval of other organizations. Any changes to the channel **structure** need to be approved by a set of organizations that can satisfy the channel policies.
+
+Policies also govern the **processes** of how users interact with the channel, such as the set of organizations that need to approve a chaincode before it can be deployed to a channel or which actions need to be completed by channel administrators.
+
+Channel policies are important enough that they need to be discussed in their own topic. Unlike other parts of the channel configuration, the policies that govern the channel are determined by how different sections of the `configtx.yaml` file work together. While channel policies can be configured for any use case with few constraints, this topic will focus on how to use the default policies provided by Hyperledger Fabric. If you use the default policies used by the Fabric test network or the [Fabric sample configuration](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/configtx.yaml), each channel you create will use a combination of signature policies, ImplicitMeta policies, and Access Control Lists to determine how organizations interact with the channel and agree to update the channel structure. You can learn more about the role of policies in Hyperledger Fabric by visiting the [Policies concept topic](../policies.html).
+
+## Signature policies
+
+By default, each channel member defines a set of signature policies that references their organization. When a transaction or channel update request is submitted to the peer and ordering nodes on the channel channel, the nodes read the signatures attached to the transaction and evaluate them against the signature policies defined in the channel configuration. Every signature policy has rule that specifies the set of organizations and identities whose signatures can satisfy the policy. You can see the signature policies defined by Org1 in the **Organizations** section of `configtx.yaml` below:
```

Comment:
Typo: "channel channel"

> When a transaction or channel update request is submitted to the peer and ordering nodes 

Maybe: When a proposal is submitted to a peer, or a transaction is submitted to the ordering nodes

My concern is confusion caused by this sentence implying that config updates are submitted to peers -- they are not.

Reply:
``channel channel``
Removed.

Reply:
Going to use your suggestion: When a proposal is submitted to a peer, or a transaction is submitted to the ordering nodes.